src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -122,36 +129,40 @@ public void execute_duplicateStudySpotFilteredList_failure() {
     @Test
     public void execute_invalidStudySpotIndexUnfilteredList_failure() {
         Index outOfBoundIndex = Index.fromOneBased(model.getFilteredStudySpotList().size() + 1);
```

Comment:
We're shifting from Indexing to search by name, so can remove bits of code like this.